### DISCUSSION TITLE
What does the `expires_in` in the Poll object in `ScheduledStatus` indicate?

### DISCUSSION THREAD

**Discussion by myConsciousness** [mastodon/mastodon#23633] [Q&A] [answered]
What does the `expires_in` in the Poll object in `ScheduledStatus` indicate? Am I correct in interpreting that the poll will be closed after the `scheduled_at` + `expires_in` seconds?

- https://docs.joinmastodon.org/entities/ScheduledStatus/#params-poll-expires_in

**Comment by Gargron** [comment, marked as answer]
Yes, I think so.

**Reply by myConsciousness** [reply]
Hi @Gargron ,

Thanks for your quick reply, I appreciate it! 🤝
